programmers/remove-smallest-number.js

```diff
@@ -0,0 +1,11 @@
+function solution(arr) {
+    if (arr.length === 1) {
+        return [-1];
+    } else {
+        arr.indexOf(Math.min.apply(null,arr));
+        arr.splice(arr.indexOf(Math.min.apply(null,arr)),1);
+        return arr;
+    }
+}
```

Comment:
if문을 이용하셔서 길이가 1인 경우를 처리하셨는데 삼항연산자를 이용하면 더 깔끔한 코드가 됩니다.
전개 연산자를 이용하면 apply를 이용안하셔도 됩니다.
filter() 메소드를 이용하면 쉽게 제일 작은 수를 제거할수 있습니다.

```suggestion
function solution(arr) {
    const min = Math.min(...arr);
    const answer = arr.filter(num => num > min);
    return arr.length === 1 ? [-1] : answer;
}
```